src/components/tx/modals/TokenTransferModal/SendAssetsForm.tsx

```diff
@@ -67,6 +67,7 @@ const SendAssetsForm = ({ onSubmit, formData }: SendAssetsFormProps): ReactEleme
   const formMethods = useForm<SendAssetsFormData>({
     defaultValues: { ...formData, [SendAssetsField.type]: SendTxType.multiSig },
     mode: 'onChange',
+    delayError: 500,
```

Comment:
@iamacook FYI this worked for other fields, but not for the address input. So I had to add a manual debounce.